ci_framework/roles/run_hook/tasks/playbook.yml

```diff
@@ -73,7 +74,7 @@
           -e "hook_name={{ hook_name }}"
           -e "playbook_dir={{ playbook_path | dirname }}"
           {{ playbook_path }}
-        create: "{{ hook.creates | default(omit) }}"
```

Comment:
@cjeanner @pablintino the build is giving error because is trying to execute `noop.yml.sh` (because is transfering `hooks/playbooks/noop.yml`) since I changed the `create` parameter that was wrong. 

Is this parameter necessary? If not I can delete it. Thx.

Reply:
`creates` parameter allows to ensure we don't run the same hook twice by allowing to check for the existence of some location. We therefore should keep it, in order to ensure idempotent runs (or, at least, try to)